vc/vc.go

```diff
@@ -33,6 +34,103 @@ func VCContextV1URI() ssi.URI {
 	}
 }
 
+const (
+	// JSONLDCredentialProofFormat is the format for JSON-LD based credentials.
+	JSONLDCredentialProofFormat string = "ldp_vc"
+	// JWTCredentialsProofFormat is the format for JWT based credentials.
+	JWTCredentialsProofFormat = "jwt_vc"
```

Comment:
Maybe add a note that this is currently 'our' const definition and that various specs have not yet decided on a single const?